packages/ui/src/icons/logos/logo-mpc-portal.native.tsx

```diff
@@ -1,12 +1,15 @@
 import { Component, forwardRef } from 'react';
+import { SvgProps } from 'react-native-svg';
 
 import LogoMpcPortalLogo from '../../assets/icons/logos/logo-mpc-portal-24-24.svg';
-import { Icon, IconProps } from '../icon/icon.native';
+import { Icon } from '../icon/icon.native';
 
-export const LogoMpcPortal = forwardRef<Component, IconProps>(({ variant, ...props }, ref) => {
+export const LogoMpcPortal = forwardRef<Component, SvgProps>((props, ref) => {
   return (
     <Icon ref={ref} {...props}>
       <LogoMpcPortalLogo />
     </Icon>
   );
 });
+
+LogoMpcPortal.displayName = 'LogoMpcPortal';
```

Comment:
is there a plugin or build tool of some sort that does this automatically?

Reply:
There isn't a `--fix` for this rule, but I vaguely remember seeing a babel plugin, I can look into that. Either way this isn't a particularly annoying rule, display names are only required for `forwardRef`s, and we only have a handful of those, mostly in the ui library. 
The repetitive icon definitions should be abstracted away with a HOC (I'm slowly working on this as part of icon system revamp).
cc @edgarkhanzadian